elements/lisk-transaction-pool/test/unit/transaction_pool.spec.ts

```diff
@@ -358,14 +353,14 @@ describe('TransactionPool class', () => {
 			// Arrange
 			const MAX_TRANSACTIONS = 10;
 			transactionPool = new TransactionPool({
-				applyTransactions: jest.fn(),
+				verifyTransaction: jest.fn(),
 				minEntranceFeePriority: BigInt(10),
 				maxTransactions: MAX_TRANSACTIONS,
 				maxPayloadLength: 15360,
 			});
 
 			const tempApplyTransactionsStub = jest.fn();
```

Comment:
also in other places below
```suggestion
			const tempVerifyTransactionsStub = jest.fn();
```